automation/prod-workflow-inc.sh

```diff
@@ -42,6 +44,7 @@ getAccessToken() {
 
   if [ "${NEED_TOKEN}" = "true" ]
   then
+    printf "\nRetrieving new ACCESS_TOKEN for user '%s'" "${user}"
```

Comment:
in `automation/prod-workflow-inc.sh`-based scripts this is all already printed due to having `set -x` set (ie "debug printing"). Might be another alternative to set the same option in `automation/submission-perf-inc.sh`?

Reply:
Maybe add `set -x` in `submission-perf-inc.sh` as well? But I am a little hesitant considering it is mainly used in perf tests and not by us. I can then remove this println from `prod-workflow-inc.sh` but keep it in `submission-perf-inc.sh`. How does that sound?

Reply:
Sounds good to me